src/calendar-list/index.tsx

```diff
@@ -348,10 +354,10 @@ class CalendarList extends Component<CalendarListProps, CalendarListState> {
           getItemLayout={this.getItemLayout}
           onViewableItemsChanged={this.onViewableItemsChanged}
           viewabilityConfig={this.viewabilityConfig}
-          initialScrollIndex={this.state.openDate ? this.getMonthIndex(this.state.openDate) : false}
+          initialScrollIndex={this.state.openDate ? this.getMonthIndex(this.state.openDate) : undefined}
           showsVerticalScrollIndicator={showScrollIndicator}
           showsHorizontalScrollIndicator={horizontal && showScrollIndicator}
-          testID={testID}
+          testID={this.props.testID}
```

Comment:
Why this change? 
why not destructing it from `this.props` as it was..

Reply:
Just to match other props we're not manipulating before passing

Reply:
If anything I would move all `this.props.xxx` to the destructor so it's unified.. no?

Reply:
I'm hoping to extract FlatList props and pass them as a buldle and not one by one. This is how it's written all over the lib, I was just conforming, it's not a bit issue